config/authors/shloka-bhalgat-unskript.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "shloka-bhalgat-unskript",
+  "bio": "",
```

Comment:
Senior Member of Technical Staff at unSkript, Inc

Reply:
Updated @5wathi